dependencymods.md

```diff
@@ -25,6 +27,8 @@ Some of the core features are:
   - This abstracts code used to have global and save settings.
 - ModInterOp
   - This allows mods to register methods with a name so other mods can call them without having a hard dependency.
+  - This is superseeded by MonoMod.ModInterop
```

Comment:
```suggestion
  - This is superceded by MonoMod.ModInterop
```

Reply:
[i believe that's meant to spell superseded then?](https://writingexplained.org/supercede-vs-supersede-difference)